src/autocomplete/docs/index.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import Box from 'ui-box'
+import starWarsNames from 'starwars-names'
+import Autocomplete from '../src/Autocomplete'
+import { TextInput } from '../../text-input'
+import { Button } from '../../buttons'
+
+// Import main component as 'raw' text to be parsed with an AST to get get propTypes information
+/* eslint-disable import/no-unresolved, import/no-webpack-loader-syntax */
+import autocompleteSource from '!raw-loader!../src/Autocomplete'
+/* eslint-enable import/no-unresolved, import/no-webpack-loader-syntax */
+
+/**
+ * Code examples
+ */
+import exampleDefaultBasic from './examples/default-basic.example'
+import exampleDefaultWithProp from './examples/default-with-children-prop.example'
+import exampleonFocusBasic from './examples/onFocus-basic.example'
+import examplefilterDisabledBasic from './examples/filterDisabled-basic.example'
+import examplewithTitleBasic from './examples/withTitle-basic.example'
+import exampleWithButtonBasic from './examples/withButton-basic.example'
+import exampleWithManyOptions from './examples/allOptions.example'
+
+// Generate a big list of items
+const items = [
+  ...starWarsNames.all,
+  ...starWarsNames.all.map(x => `${x} 2`),
+  ...starWarsNames.all.map(x => `${x} 3`)
+].sort((a, b) => {
+  const nameA = a.toUpperCase()
+  const nameB = b.toUpperCase()
+  if (nameA < nameB) {
+    return -1
+  }
+  if (nameA > nameB) {
+    return 1
+  }
+  return 0
+})
+
+const title = 'Autocomplete'
+const subTitle = 'A component to filter through a list of options'
+
+const implementationDetails = (
+  <div>
+    <p>
+      The <code>Autocomplete</code> components renders a filterable list of
+      options in a Popover — it does not render the text input - If you need a
```

Comment:
Being a type nazi here. hyphen (-) is not an em dash (—). Please use a em dash in this case. Learn more here:  https://practicaltypography.com/hyphens-and-dashes.html

Reply:
Note that because atom uses a monospace font, it doesn't look any different in the code editor.